### PR TITLE
Update design of `InsurableLimits` component

### PR DESCRIPTION
**Pull request by robinandeer**
## Describe your changes

Tweak the design of `InsurableLimits` component to better match design.

Improve typing of "radius" theme helper.

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by robinandeer** [comment]
Current dependencies on/for this PR:
* main
  * **PR #888** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/888" target="_blank">Graphite</a> 
    * **PR #890** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/890" target="_blank">Graphite</a> 
      * **PR #891** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/891" target="_blank">Graphite</a> 
        * **PR #892** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/892" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/892?utm_source=stack-comment).

File: packages/ui/src/lib/theme/radius.ts [hunk 1]
```diff
@@ -1,16 +1,17 @@
...
-  2: '16px',
-  3: '20px',
-  4: '28px',
+const base = {
+  0: 8,
+  1: 12,
+  2: 16,
+  3: 20,
+  4: 28,
 }
```
Comment (KarnellSchultz) [inline packages/ui/src/lib/theme/radius.ts:7]:
```suggestion
} as const
```

By adding `as const` here we're changing the type of the `base` value from `number` to the actual value, for example `8` or `28`. 

Also noticed that we don't need to add `px` value for border radius 👍

**Review by KarnellSchultz — APPROVED** [review]

File: packages/ui/src/lib/theme/radius.ts (hunk 1 above)
Comment (gustaveen) [reply packages/ui/src/lib/theme/radius.ts:7]: I just thought it's nice to be explicit but I'm all good if you think it's clear enough to without `px` :)

**Comment by robinandeer** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/892?utm_source=stack-comment).


Job Reference: KrNNugJV01lmDDBZCLzA

**Pull request merged**